integration-tests/src/test/java/com/datastax/oss/driver/osgi/OsgiBaseIT.java

```diff
@@ -78,7 +78,7 @@ public DriverConfigLoader configLoader() {
   @SuppressWarnings("unchecked")
   public void should_connect_and_query() {
     SessionBuilder<CqlSessionBuilder, CqlSession> builder =
-        SessionUtils.baseBuilder()
+        CqlSession.builder()
```

Comment:
I prefer not to use our test infra inside this test to stay close to what users would actually do in their OSGi applications. We never know if by using the test infra we could introduce a biais (e.g. classes being loaded by the wrong classloader, etc.).

Reply:
I think this is fine, we will probably want to have a separate test set up anyways if we are using different session types as different jars will likely be required.

Reply:
Exactly. These tests have been failing consistently when ran against the DSE driver because, as you guessed, they rely on specific jars placed in specific folders on the filesystem. We will have to mirror these tests in the DSE driver test suite anyway.